data/ecosystems/s/solady.toml

```diff
@@ -0,0 +1,10 @@
+# Ecosystem Level Information
+title = "Solady"
```

Comment:
Could you also add Solady as a sub_ecosystem of Ethereum and remove the repo link from Ethereum?